data/json/items/armor/power_armor.json

```diff
@@ -131,15 +131,15 @@
     "id": "power_armor_helmet_basic",
     "type": "ARMOR",
     "category": "armor",
-    "name": "basic power armor helmet",
-    "description": "A basic helmet, designed for use with the DoubleTech Power Armor, Mk. I.  Offers excellent protection from both attacks and environmental hazards.  Like all DoubleTech power armor, the control and climate-regulation systems require direct skin contact.",
+    "name": "environmental combat helmet",
+    "description": "A fully enclosed combat helmet for hazardous environments, this was designed to fit with a powered exoskeleton.  In actual practice, most soldiers in the field just wore normal headgear due to the visibility restrictions of the full helmet.  The environmental controls function best with direct-skin contact.",
```

Comment:
The army would almost certainly require you to wear the helmet if you were wearing the power armor, otherwise it spends millions turning the soldier into a walking tank only to leave the soldier fully vulnerable to both well aimed bullets and explosives/fragmentation.

Perhaps replace it with an excerpt saying the reduced visibility was a large downside in urban combat or other flavor text.